packages/components/src/internal/components/editable/actions.ts

```diff
@@ -31,45 +21,33 @@ import {
     CellValues,
     EditableColumnMetadata,
     EditableGridLoader,
-    EditableGridModels,
     EditorMode,
-    EditorModel,
-    EditorModelAndGridData,
-    EditorModelUpdates,
+    EditorModel, EditorModelProps,
     MessageAndValue,
     ValueDescriptor,
 } from './models';
 
 import { decimalDifference, genCellKey, getLookupFilters, getValidatedEditableGridValue, parseCellKey } from './utils';
 
-const EMPTY_ROW = Map<string, any>();
-let ID_COUNTER = 0;
-
 /**
- * @deprecated Use initEditableGridModel() or initEditableGridModels() instead.
- * This method does not make use the grid loader paradigm. Usages of this method directly
- * is susceptible to data misalignment errors with the associated view.
+ * Do not use this method directly, use initEditorModel instead
  */
-export const loadEditorModelData = async (
-    queryModelData: Partial<QueryModel>,
-    editorColumns?: QueryColumn[],
+const loadEditorModelData = async (
+    orderedRows: string[],
+    rows: Record<string, any>,
+    columns?: QueryColumn[],
     forUpdate?: boolean
+    // TODO: When all non EditorModel.init usages of this are gone update this to only return cellValues and move it to
```

Comment:
I am confused by this TODO as it looks like there is only one usage of this method now and it is within this same file

Reply:
This was one of the first TODOs I added, and am still planning on moving this to the EditorModel, but want to get the other parts of this branch stabilized first.